api/Permissions.json

```diff
@@ -385,6 +385,54 @@
           }
         }
       },
+      "gamepad_permission": {
+        "__compat": {
+          "description": "<code>gamepad</code> permission",
+          "support": {
+            "chrome": {
+              "version_added": "86"
```

Comment:
Note below, that the corresponding directive for this permission is behind a flag. It is not clear to me if this is behind a flag as well, but it doesn't look like it on code inspection - i.e. you can check for this permission (not tested)

Reply:
Gamepad permission not present for this API - as noted in https://bugzilla.mozilla.org/show_bug.cgi?id=1704005#c30